src/pages/entry.js

```diff
@@ -1,19 +1,53 @@
-import React from 'react'
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import React, { Fragment } from 'react'
 import { withRouter } from 'react-router-dom'
+import styles from './entry.styles'
```

Comment:
Is it ok to add styles to pages? I didn't see any other pages with styles.

Reply:
It seems like the more standard practice is to keep these styles in this same file (by "standard practice" I mean that I saw it in one other place) 🤷‍♂.

Reply:
I usually like to keep styles on the same file.

I'm also against default exports, but it's more of a personal preference (importing a default prevents us from renaming it across easily, but we're not using TypeScript so that's fine).